experimental/s2ef_mapping.py

```diff
@@ -0,0 +1,51 @@
+from tqdm import tqdm
+import glob
+import ase.io
+import pickle
+import os
+import multiprocessing as mp
+import numpy as np
+
+
+def get_mapping(system):
+    randomid = os.path.basename(system)[:-7]
+    try:
+        old_images = ase.io.read(system, ":")
+        old_pos = [image.get_positions() for image in old_images]
+
+        new_images = ase.io.read(f"/home/jovyan/projects/ocp/data/trajs/train_02_01/{randomid}.traj", ":")
+        new_pos = [image.get_positions() for image in new_images]
+
+        dict_mapping = {}
+        # maps old frame idx -> new frame idx
+        for old_idx, pos in enumerate(old_pos):
+            for new_idx, pos_2 in enumerate(new_pos):
+                if (pos == pos_2).all():
+                    dict_mapping[old_idx] = new_idx
+                    break
```

Comment:
maybe we should also add an assert that all the keys are values are unique ?

Reply:
Something like:
len(set(dict_mapping.keys())) == len(set(dict_mapping.values()))